### ISSUE TITLE
Add more information about tagged types in the Encoding pretty printer

### ISSUE THREAD

**Issue by axman6** [well-typed/binary-serialise-cbor#72]
As I noted in #69, it would be nice to add some more info to the pretty printer about the content of tags, so we can add notes such as `# tag(0)[time string]`, `# tag(64)[uint8 Typed Array]` (see #62 and #63). I'm happy to do this one day when we support more of the extensions.

**Comment by dcoutts** [comment]
Right, sounds sensible. I think we'd just need a built-in mapping of known tags.

**Comment by sopvop** [comment]
IANA registry for [reference](http://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml)

**Comment by bgamari** [comment]
Patches welcome! 😄
